src/bp-templates/bp-nouveau/buddypress/document/index.php

```diff
@@ -40,7 +40,7 @@
 		<?php
 		bp_get_template_part( 'common/search-and-filters-bar' );
 
-		if ( ! empty( $active_extensions ) && bp_is_profile_document_support_enabled() && is_user_logged_in() ) {
+		if ( ! empty( $active_extensions ) && bp_is_profile_document_support_enabled() && is_user_logged_in() && apply_filters( 'bp_user_can_create_document', true ) ) {
```

Comment:
@chetansatasiya  Not mandatory but just check if we can do that.

`apply_filters( 'bp_user_can_create_document', true )` if we using same filter multiple place can't we create function for that and use it. Like BuddyPress doing that.